client/blocks/reader-share/social.jsx

```diff
@@ -14,17 +14,17 @@ import { recordReaderTracksEvent } from 'calypso/state/reader/analytics/actions'
  * Local variables
  */
 const actionMap = {
-	twitter( post ) {
+	x( post ) {
 		const baseUrl = new URL( 'https://twitter.com/intent/tweet' );
```

Comment:
```suggestion
		const baseUrl = new URL( 'https://x.com/intent/tweet' );
```

Reply:
How about we add a comment for this in codebase?

Reply:
Done!